app/models/forms/contributor.rb

```diff
@@ -0,0 +1,99 @@
+module Forms
+  class Contributor < Base
+    # Indicates that this is a deposit, and therefore should be fully validated.
+    attr_accessor :_deposit
+    attr_accessor :id
+    attr_accessor :first_name
+    attr_accessor :last_name
+    attr_accessor :full_name
+    attr_accessor :with_orcid # radio button
```

Comment:
Here is an example of where the form object reflects the form that is rendered. In reform, this would have been a virtual attribute.